pkgs/stdenv/darwin/default.nix

```diff
@@ -969,7 +972,7 @@ in
         sed serf sharutils sqlite subversion sysctl texinfo unzip which xz zstd
 
         # CF dependencies - don’t rebuild them.
-        icu libiconv libxml2 zlib;
+        icu libiconv libiconv-darwin libxml2 zlib;
```

Comment:
It might be possible to inherit only libiconv-darwin. The intent is both refer to the same path. If I just inherit libiconv, libicon-darwin would refer to the one from the current stage instead of the previous one.

Reply:
Is this so packages used in stdenv that happen to directly refer to libiconv-darwin rather than libiconv don't use the one from the current stage?

Reply:
I wasn’t think of that, but yeah. Nothing _should_, but there’s no way of stopping packages from doing it anyway. If they do, it should be the same package for both. (I wish there were a better way of handling aliases like this.)